husarion_ugv_description/urdf/panther/body.urdf.xacro

```diff
@@ -6,30 +6,31 @@
     name="body"
     params="wheel_radius imu_xyz imu_rpy">
 
-    <link name="base_link" />
+    <link name="base_footprint" />
 
     <joint name="base_to_body_joint" type="fixed">
       <origin xyz="0.0 0.0 ${wheel_radius}" rpy="0.0 0.0 0.0" />
-      <parent link="base_link" />
-      <child link="body_link" />
+      <parent link="base_footprint" />
+      <child link="base_link" />
     </joint>
```

Comment:
I'll just mention that `base_link` should be the main link of the robot and a parent to all other links. base_footprint should just be a helper link for visualization/navigation. Btw, check also joint names